hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaTargetFilterQueryManagement.java

```diff
@@ -72,7 +71,7 @@
 public class JpaTargetFilterQueryManagement implements TargetFilterQueryManagement {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(JpaTargetFilterQueryManagement.class);
-    
+
```

Comment:
please check if you still need the `findDistributionSetAndThrowExceptionIfNotFound` method and its usage under `findByAutoAssignDSAndRsql`